docs/getting-started/continuous_integration.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: default
+title: Continuous Integration
+parent: Getting started
+nav_order: 5
+permalink: /getting-started/continuous-integration/
+---
+
+# Continuous Integration
+
+OSS-Fuzz offers Continuous Integration(CI) support for projects hosted on
```

Comment:
Let's make this shorter.

`OSS-Fuzz offers CIFuzz, which will run your fuzz targets each time a pull request is submitted, for projects hosted on GitHub. This allows you to detect and fix bugs before they are submitted.`

Reply:
Done.